views/topic/show.ejs

```diff
@@ -86,7 +86,7 @@
         <pre><code class="R"><%- topic.examples %></code></pre>
       </section>
       <% if(inViewerPane){ %>
-        <a href="javascript:;" onclick="window.runExamples();" class="btn btn-primary">Run examples</a>
+        <a href="www.google.com" onclick="window.runExamples()" id="js-examples" class="btn btn-primary">Run examples</a>
```

Comment:
should be fixed : https://github.com/datacamp/rdocumentation-app/commit/6d7ecc31a3141265e9c7bfe42e103e6e8e953327